src/pages/Homepage.js

```diff
@@ -19,31 +36,53 @@ const Homepage = () => {
     }
   };
 
+  function updateForecastDegree(event){
+    setDegreeType({
+      degreeType: event.target.value
+    })
+    console.log(setDegreeType)
```

Comment:
console debuggers should not be part of your PRs, it's best practice to remove all your debuggers in your PRs so they don't accidentally ship to production.

NB: there's a better way to handle debugging in JS outside of logging to the console. Note it down and we can discuss over the weekend as well.